### PR TITLE
Add a license to the project.

### PR DESCRIPTION
**Pull request by anubhavcodes**
This is one of the requirement for the package to be accepted as a brew package. I will update the PR link here that will be made in the homebrew-core repo in a moment.

### CODE REVIEW

**Comment by anubhavcodes** [comment]
https://github.com/Homebrew/homebrew-core/pull/63624

Here is the PR to merge the homebrew formula

**Comment by anubhavcodes** [comment]
Let's close this PR as it doesn't contribute much to the project and we are not going ahead with `brew install`.

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License
+
+Copyright (c) 2012-2020 Pivotal Software, Inc. All Rights Reserved.
```
Comment (jadolg) [inline LICENSE:3]: Just remove this line and I'll approve it ;)

**Review by jadolg — CHANGES_REQUESTED** [review]

File: LICENSE (hunk 1 above)
Comment (anubhavcodes) [reply LICENSE]: Done.

**Pull request merged**